### ISSUE TITLE
If no camera exists in the scene, MRTK Configure doesn't properly set up the scene

### ISSUE THREAD

**Issue by Ecnassianer** [microsoft/MixedRealityToolkit-Unity#3403]
## Overview

If there is no camera in the scene (say for instance, if you screwed up your MixedRealityPlayspace and want to start over with a fresh one), then the Mixed Reality Toolkit Configure option won't create a playspace or a new camera.

## Expected Behavior

MRTK can create the necessary camera it needs even if one doesn't exist.

## Actual Behavior
No MixedRealityPlayspace is created

## Steps to reproduce
Install MRTK
Create a new scene.
Delete the camera
From the dropdown menus choose Mixed Reality Toolkit->Configure

## Unity Editor Version
2018.3.0f2

## Mixed Reality Toolkit Release Version
Whatever I cloned on 2/3/19

**Comment by markgrossnickle** [comment]
Reproduced in 2018.3.6f1

I discovered this because it was modifying a camera I wanted to keep so I temporarily removed that camera from the scene to hide it from the script, but the script fails without a camera being found. 

I'm not a big fan of it silently changing the camera like that (the camera was hidden in the hierarchy so I didn't even know it happened at first).  I do realize you also don't want two cameras in the scene so I get why it does it.
